test/test_dot_device.cc

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2017-2022, University of Tennessee. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include "test.hh"
+#include "cblas_wrappers.hh"
+#include "lapack_wrappers.hh"
+#include "blas/flops.hh"
+#include "print_matrix.hh"
+
+// -----------------------------------------------------------------------------
+template< typename Tx, typename Ty >
+void test_dot_device_work( Params& params, bool run )
+{
+    using namespace testsweeper;
+    using namespace blas;
+    typedef scalar_type<Tx, Ty> scalar_t;
+    typedef real_type<scalar_t> real_t;
+    typedef long long lld;
```

Comment:
`typedef` changes to `using`. Remove `lld`.

Reply:
Do I need to change to using? I reviewed other testers and axpy for example, uses llong but doesn't have using long long at the top. It is defined test.hh which is included at the beginning.

Reply:
Just match what other testers are doing. I think remove `using namespace blas`, which may require adding `blas::` in some places. Don't need `using llong` because that's in test.hh, as you noticed. Otherwise:
```
    using scalar_t = blas::scalar_type< TX, TY >;
    using real_t   = blas::real_type< scalar_t >;
```